primitives/runtime/src/transaction_validity.rs

```diff
@@ -53,7 +53,8 @@ pub enum InvalidTransaction {
 	/// itself. As the verifying side does not know which additional data was used while signing
 	/// it will only be able to assume a bad signature and cannot express a more meaningful error.
 	BadProof,
-	/// The transaction birth block is ancient.
+	/// The transaction birth block is ancient. For `FRAME`-based runtimes this means:
+	/// current block number - `Era::birth` block number > `BlockHashCount`.
```

Comment:
Is it worthwhile to add an example using the polkadot defaults? Something like this: "E.g. A Polkadot mortal extrinsic included in block 1337 is valid until block 1337 + 2400."